docs/data/real_property.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
@kasao10 can you remove one of these two lines with `---` in them? proper format is:

```
---
datatype: value
otherdata: value
---
```

This was my mistake in the template that had the duplicate one